internal/views/templates/logout.html

```diff
@@ -0,0 +1,66 @@
+{{- /*
+Copyright 2024 - Swiss Data Science Center (SDSC)
+A partnership between École Polytechnique Fédérale de Lausanne (EPFL) and
+Eidgenössische Technische Hochschule Zürich (ETHZ).
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License
+*/ -}}
+{{- define "logout" -}}
+<!DOCTYPE html>
+<html lang="en" >
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="x-ua-compatible" content="ie=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <meta http-equiv="refresh" content="5; url={{.redirectURL}}" />
+        <link rel="stylesheet" href="{{.renkuBaseURL}}/static/public/theme.css">
+        <title>Renku - login</title>
+        <script type="text/javascript">
+            setTimeout(function(){ window.location.href = "{{.redirectURL}}"; }, 20000);
```

Comment:
question: shouldn't this be 5000 since the text below says "5 seconds"?

Reply:
I simply ported the existing html pages which existed in the python gateway. The refresh should come from the `meta` tag above.